js/views/itemEditVw.js

```diff
@@ -217,19 +220,33 @@ module.exports = Backbone.View.extend({
   },
 
   deleteImage: function(e) {
-    var imgIndex = $(e.target).closest('.itemImg').data('index');
+    var imageUploadArray, imgIndex = $(e.target).closest('.itemImg').data('index');
```

Comment:
Just want to make sure this is an agreed upon move because it changes the approach and coding style.